internal/provider/waypoint/resource_waypoint_application.go

```diff
@@ -187,6 +190,31 @@ func (r *ApplicationResource) Schema(ctx context.Context, req resource.SchemaReq
 					},
 				},
 			},
+			"output_values": schema.ListNestedAttribute{
```

Comment:
Same question about why we need this when https://developer.hashicorp.com/terraform/language/values/outputs already exists in Terraform